src/oversampling_methods/smotenc/interface_mlj.jl

```diff
@@ -61,11 +61,12 @@ MMI.metadata_model(
     SMOTENC,
     input_scitype = Union{
         Table(Union{Infinite, Finite}),
+        AbstractVector
     },
     output_scitype = Union{
         Table(Union{Infinite, Finite}),
+        AbstractVector
     },
```

Comment:
Why is this one not repaired in the same pattern as the others? Don't you need `Tuple` somewhere?